frontend/src/TooljetDatabase/Forms/EditColumnForm.jsx

```diff
@@ -0,0 +1,227 @@
+        column_name: selectedColumn?.Header,
+        data_type: selectedColumn?.dataType,
+        ...(columnName !== selectedColumn?.Header ? { new_column_name: columnName } : {}),
+        ...(defaultValue?.length > 0 ? { column_default: defaultValue } : {}),
+        ...(nullValue !== isNotNull
+          ? {
+              constraints_type: {
+                is_not_null: isNotNull,
+              },
+            }
+          : {}),
+      },
+    };
+
+    if (columnName !== selectedColumn?.Header || defaultValue?.length > 0 || nullValue !== isNotNull) {
+      setFetching(true);
+      const { error } = await tooljetDatabaseService.updateColumn(organizationId, selectedTable.table_name, colDetails);
+      setFetching(false);
+      if (error) {
+        toast.error(error?.message ?? `Failed to edit a column in "${selectedTable.table_name}" table`);
+        onClose && onClose();
+        return;
+      }
+    }
+    tooljetDatabaseService.viewTable(organizationId, selectedTable.table_name).then(({ data = [], error }) => {
+      if (error) {
+        toast.error(error?.message ?? `Error fetching columns for table "${selectedTable}"`);
+        return;
+      }
+
+      if (data?.result?.length > 0) {
+        setColumns(
+          data?.result.map(({ column_name, data_type, keytype, ...rest }) => ({
+            Header: column_name,
+            accessor: column_name,
+            dataType: data_type,
+            isPrimaryKey: keytype?.toLowerCase() === 'primary key',
+            ...rest,
+          }))
+        );
+      }
+    });
+    toast.success(`Column edited successfully`);
+    onClose && onClose();
+  };
+
+  // const handleCreate = async () => {
+  //   if (isEmpty(columnName)) {
+  //     toast.error('Column name cannot be empty');
+  //     return;
+  //   }
+  //   if (isEmpty(dataType?.value)) {
+  //     toast.error('Data type cannot be empty');
+  //     return;
+  //   }
+  // setFetching(true);
+  // const { error } = await tooljetDatabaseService.updateColumn(
+  //   organizationId,
+  //   selectedTable.table_name,
+  //   columnName,
+  //   dataType?.value,
+  //   defaultValue
+  // );
+  // setFetching(false);
+  // if (error) {
+  //   toast.error(error?.message ?? `Failed to create a new column in "${selectedTable.table_name}" table`);
+  //   return;
+  // }
+  // toast.success(`Column created successfully`);
+  //   onCreate && onCreate();
+  // };
+
+  return (
+    <div className="drawer-card-wrapper ">
+      <div className="drawer-card-title ">
+        <h3 className="" data-cy="create-new-column-header">
+          Edit column
+        </h3>
+      </div>
+      <div className="card-body">
+        <div className="edit-warning-info mb-3">
+          <div className="edit-warning-icon">
+            <WarningInfo />
+          </div>
+          <span className="edit-warning-text">
+            Editing the column could break queries and apps connected with this table. Please wait for ToolJet to update
```

Comment:
Can remove `Please wait for ToolJet to update..` we are not implementing this atm